packages/terra-arrange/docs/Arrange.md

```diff
@@ -0,0 +1,9 @@
+| Prop Name | Type | Is Required | Default Value | Description |
```

Comment:
What all do we plan on using the cli for? Is the output designed to be used with normal documentation, or is this cli just for showing a page on github?